src/components/Layout/Styles/dark.scss

```diff
@@ -0,0 +1,30 @@
+.layout {
+    &-dark {
```

Comment:
I see... so we are going to support a dark mode and light mode for any theme in general it seems. Are we doing this work in other components already?

Reply:
Not yet as this work needs to be done. Lots of work to do to support themes and i18n